src/vs/code/node/cliVerboseLogger.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ChildProcess } from 'child_process';
+import { watch, writeFileSync, unlinkSync } from 'fs';
+import { Promises } from 'vs/base/node/pfs';
+
+/**
+ * This class provides logging for verbose commands on Mac
+ * as part of #102975
+ */
+export class CliVerboseLogger {
+	constructor(readonly filename: string) {
+		writeFileSync(filename, '');
```

Comment:
I would think this code should live outside where the logger is called from because imho the logger itself does not really own the file name it gets passed into.

Reply:
I'll add the writeFileSync calls back to `cli.ts`